common/encfstools.py

```diff
@@ -279,12 +283,13 @@ def __init__(self, cfg = None, profile_id = None, mode = None, parent = None,*ar
             self.mode = self.config.get_snapshots_mode(self.profile_id)
 
         self.parent = parent
+        self.read_only = read_only
         self.args = args
         self.kwargs = kwargs
 
-        self.ssh = sshtools.SSH(*self.args, symlink = False, **self.split_kwargs('ssh'))
-        self.rev_root = EncFS_mount(*self.args, symlink = False, **self.split_kwargs('encfs_reverse'))
-        super(EncFS_SSH, self).__init__(*self.args, **self.split_kwargs('encfs'))
+        self.ssh = sshtools.SSH(*self.args, symlink = False, read_only = self.read_only, **self.split_kwargs('ssh'))
+        self.rev_root = EncFS_mount(*self.args, symlink = False, read_only = self.read_only, **self.split_kwargs('encfs_reverse'))
```

Comment:
Sounds good! I wasn't sure how paranoid you wanted to be about ro on these bits :) Adjusting.